content/docs/tutorials/scrape_paginated_sites.md

```diff
@@ -257,13 +267,16 @@ const { min, max } = getFiltersFromUrl(request.url);
 const newFilters = splitFilter({ min: min * 100, max: max * 100 });
 
 // And we just enqueue those 2 new filters so the process will recursively repeat until all pages get to the PAGINATION phase
+const requestsToEnqueue = [];
 for (const filter of newFilters) {
-    await requestQueue.addRequest({
+    await requestsToEnqueue.push({
```

Comment:
extra await?

Reply:
good catch, I thought I removed it, missed this one